### PR TITLE
IR-5292 Add asset drag and drop to image link

### PR DESCRIPTION
**Pull request by AidanCaruso**
## Summary
Image links don't allow for editing. This PR adds edit/asset drag and drop support.

## References
Closes https://tsu.atlassian.net/browse/IR-5292

### CODE REVIEW

File: packages/ui/src/components/editor/ImageLink/index.tsx [hunk 1]
```diff
@@ -23,14 +23,17 @@ All portions of the code written by the Infinite Reality Engine team are Copyrig
...
 import React, { ImgHTMLAttributes, useEffect, useRef } from 'react'
 import { useTranslation } from 'react-i18next'
 import { twMerge } from 'tailwind-merge'
+import FileBrowserInput from '../input/FileBrowser'
 import ImageUrlFallback from './image-url-fallback.png'
 
 export interface ImageLinkProps extends Omit<ImgHTMLAttributes<HTMLImageElement>, 'onChange' | 'onBlur'> {
   variant?: 'lg' | 'md' | 'sm' | 'full'
-  onChange?: (value: string) => void
+  onRelease?: (value: string) => void
```
Comment (aditya-mitra) [inline packages/ui/src/components/editor/ImageLink/index.tsx:36]: we could keep the `onChange` prop name and pass it to FileBrowserInput
`onRelease` is more associated with sliders where we _release_ the input invoking an action

**Review by aditya-mitra — CHANGES_REQUESTED** [review]
Am unsure about file browser support in ImageLink
I have asked the same in [figma](https://www.figma.com/design/ln2VDACenFEkjVeHkowxyi?node-id=2370-20234#1014424457)

I will update this PR depending upon the designs and further resolution.

**Review by DanielBelmes — APPROVED** [review]

**Review by DanielBelmes — CHANGES_REQUESTED** [review]
Was duplicated by https://github.com/ir-engine/ir-engine/pull/610 but that PR is more preferable. Might close this soon

**Comment by aditya-mitra** [comment]
> Was duplicated by #610 but that PR is more preferable. Might close this soon

Yes Daniel you are right!
Lets close in favour of #610

**Pull request closed without merging**